common/abstracts/_variables.scss

```
@@ -38,11 +38,15 @@ $topicImageOrangeFill: rgba(255, 166, 0, 0.25);
 $topicImageWhiteStroke: rgba(255, 255, 255, 1);
 $topicImageWhiteFill: rgba(255, 255, 255, 0.25);
 
+$topicImageBlackTransparentStroke: rgb(0, 0, 0);
+$topicImageBlackTransparentFill: rgb(0, 0, 0, 0);
```

use `rgba` when 4 numbers, this variable could also be changed to `none` instead of using a color value
```suggestion
$topicImageBlackTransparentFill: rgba(0, 0, 0, 0);
```